imagetest/test_suites/storage_perf/startupscripts/install_fio.ps1

```diff
@@ -0,0 +1,3 @@
+cd C:\
+gsutil cp gs://gce-image-build-resources/windows/fio-3.35-x64.msi C:\
+msiexec.exe /A "C:\fio-3.35-x64.msi" /quiet /norestart
```

Comment:
Small detail: /A shouldn't be needed since this will run as SYSTEM with administrative privileges.

You may also want to enclose the msiexec inside of a try/catch block, or use /L "file_path" to log to a file and then Get-Content the file to help surface errors.

Reply:
done

Reply:
it seems to be quite difficult to verify errors here, so I'll plan to use golang code to run these commands and print errors from there.

this script would be updated to be a clone of the golang code once that is done.

Reply:
If the script is going to call a go binary we could handle all the logging and error messages in there and pass it up instead of handling it with powershell.

Alternatively, I believe msiexec will log to Event Viewer even in this context, so if the powershell script were to retrieve the Application Event Log after attempting installation, that might also work, assuming the error in msiexec doesn't block the rest of the script (try/catch would probably be needed for that though. Might be too complex for what it needs to do)

Reply:
You could try something like we did for installing SQL but substituting msiexec:
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/image_build/sqlserver/sql_install.ps1#L286

Reply:
Just some ideas to consider. Not saying any one of these approaches is more or less correct.

Reply:
yeah at least for now, the idea is that in storage_perf_utils.go, there is some code which installs fio on linux using go code.

The startup script install_fio.sh was based on those exact go code lines, and those go lines can print the error message if any command fails.

For windows, the idea would be to have go lines just like what is in storage_perf_utils.go. This is quite useful right now as well, because it seems that some of these windows startup commands might be failing, but it's hard to debug this ps1 script directly. After the go code to install fio on windows succeeds, the ps1 script would be a direct copy of what succeeded.